### PR TITLE
add relativewave-form 1.4.0

### PR DESCRIPTION
**Pull request by varunyellina**
This is a prototyping app called Form. Distributed by RelativeWave, acquired by Google.

### CODE REVIEW

File: Casks/relativewave-form.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'relativewave-form' do
+  version '1.4.0'
+  sha256 '9369d1e468938da60fffa890b5c8d4ec2888dcde990b2d2763fb56bbae1825af'
+
+  url "https://relativewave.storage.googleapis.com/files/Form-#{version}.dmg"
+  name 'Form by RelativeWave'
```
Comment (adityadalal924) [inline Casks/relativewave-form.rb]: Should be `RelativeWave Form` (company first)

File: Casks/relativewave-form.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'relativewave-form' do
+  version '1.4.0'
+  sha256 '9369d1e468938da60fffa890b5c8d4ec2888dcde990b2d2763fb56bbae1825af'
+
+  url "https://relativewave.storage.googleapis.com/files/Form-#{version}.dmg"
```
Comment (adityadalal924) [inline Casks/relativewave-form.rb]: Needs a comment https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment

File: Casks/relativewave-form.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Form by RelativeWave'
+  homepage 'http://www.relativewave.com/form/'
+  license :unknown # TODO: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (adityadalal924) [inline Casks/relativewave-form.rb]: :gratis - as it is free-to-use, closed source

**Comment by adityadalal924** [comment]
Please modify and squash your commits: https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#squashing

**Comment by varunyellina** [comment]
Made necessary changes and squashed commits.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 6c1af93c7f02778f77e5800b8914c0a1595a62e0. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**